lib/decanter/parser.rb

```diff
@@ -2,19 +2,36 @@ module Decanter
   module Parser
 
     class << self
-      def parser_for(klass_or_sym)
-        parser_str = case klass_or_sym
+      def parsers_for(klass_or_syms)
```

Comment:
Awesome work here. It seems like the logic about how to find a class (both for parsers and decanters) is duplicated a bit throughout the gem. Something to think about refactoring at some point.